motion/vtf_guidance_and_control/scripts/guidance_and_control_node.py

```diff
@@ -0,0 +1,252 @@
+#!/usr/bin/env python
+# Written by Aksel Kristoffersen
+
+import rospy
+import numpy as np
+import tf
+import tf2_ros
+import tf2_geometry_msgs
+from tf.transformations import euler_from_quaternion, quaternion_from_euler
+from nav_msgs.msg import Odometry, Path
+from geometry_msgs.msg import Wrench, PoseStamped
+from path import Path as Path1
+from auv_model import AUVModel
```

Comment:
"Path1" as a way to avoid name overlaps with nav_msgs isn't really clean. VtfPath, OwnPath or something along those lines would be better